frontend/src/pages/home/videos/HomeVideos.tsx

```diff
@@ -43,86 +61,103 @@ const HomeVideosPage = () => {
   }, [pollName, showWarningAlert, t]);
 
   return (
-    <AlternatingBackgroundColorSectionList lastItemIsPrimary>
-      <TitleSection title={t('home.collaborativeContentRecommendations')}>
-        <Typography paragraph fontSize="1.1em">
-          {t('home.tournesolIsAParticipatoryResearchProject')}
-        </Typography>
+    <>
+      <Box padding={4} color="white" bgcolor="#1282B2">
```

Comment:
Could we define this color as a constant somewhere?  
Maybe even in "theme.ts" as a new variant `background.home` ?

Reply:
Yes it's a good idea.

I don't know how to name the they, as this blue is used on other pages like the FAQ or About.

Maybe `background.secondary`?

Reply:
Should be fixed by https://github.com/tournesol-app/tournesol/pull/1287/commits/6576a8f15fdae9de7c5aba798cd325961a63c301

I also updated all components using this color.

Reply:
"background.secondary" sounds good to me.  
Or "background.emphatic" to express that it's a flashier color compared to the default background.

Reply:
Good, I'll use `emphatic`.